src/main/java/io/deephaven/csv/containers/ByteSlice.java

```diff
@@ -110,6 +110,20 @@ public CharSequence subSequence(final int start, final int end) {
         return new ByteSlice(data, newBegin, newEnd);
     }
 
+    /**
+     * Trim the padding bytes from the front and back of the slice.
+     *
+     * @param padding The padding byte.
+     */
+    public void trimPadding(byte padding) {
```

Comment:
This appears to be unused (even when reviewing other PR).

Reply:
thanks, deleted